config/m4/module.m4

```diff
@@ -281,27 +273,48 @@ AC_DEFUN([_CHECK_MODULE_PACKAGE],
 #     - if set static lib 
 #       1. check it using package configuration (--static) and try to
 #          detect whole set of dependencies
-#       2. check it using includes/ldflags provided by user, if failed
+#       2. check it using includes/ldflags provided by user
 #     - for dynamic lib
 #       1. check for lib using declarations/link
 #       2. check for package
 #
 # Usage: UCX_CHECK_MODULE([id], [name], [prefix], [pkgname],
-#                         [[doc], [doc-lib], [doc-static or '-' to skip], [doc-failed]],
-#                         [action-if-found], [action-if-not-found],
-#                         [[headers], [decls], [includes], [libs - space separated], [funcs], [otherlibs]],
+#                         [[help], [help-lib], [help-static], [fail-msg]],
+#                         [if-found], [if-not-found],
+#                         [[headers], [decls], [includes], [libs], [funcs], [otherlibs]],
 #                         [alt-dirs])
 #
+# Parameters:
+#   id           - module ID, used to generate runtime configuration parameters --with-id
+#   name         - readable module name
+#   prefix       - variable prefix, in success variables prefix_CFLAGS and
+#                  prefix_LIBS will store module specific flags
+#   pkgname      - package name used to lookup module using pkg-config tool, optional
+#   help         - help string for module or empty argument to generate it automatically
+#   help-lib     - help string for module library dir or empty argument to generate
+#                  it automatically
+#   help-static  - help string for static module library or empty argument to
+#                  generate it automatically, if static link is not needed set it
+#                  to '-' value (see usage example)
+#   if-found     - macro called if module configuration found
+#   if-not-found - macro called if module configuration not found or user disabled module
+#   headers      - list of header files to check, coma-separated, optional
+#   decls        - declarations to check, coma-separated, optional
+#   includes     - additional include directives to check headers and declarations, optional
+#   libs         - list of libraries to check, whitespace-separated
```

Comment:
why some params are comma-separated and others are whitespace? its confusing IMO